src/eko/evolution_operator/__init__.py

```diff
@@ -185,22 +184,22 @@ def quad_ker(
     if ker_base.is_singlet:
         gamma_singlet = ad.gamma_singlet(order, ker_base.n, nf)
         # scale var A is directly applied on gamma
-        if sv_mode == sv.Modes.exponentiated.value:
+        if sv_mode == sv.Modes.exponentiated:
```

Comment:
Another suggestion on `enum`s (I'm really sorry to be annoying 😞), but the comparison should be done with `is`:
```suggestion
        if sv_mode is sv.Modes.exponentiated:
```
since the variants are singletons, so there is always a unique copy (like `None`).

Reply:
No, problem at all, your comments are always welcome :)
Unfortunately this isn't accepted because `numba` automatically cast `IntEnum`  to `int`. Maybe it will work if the signatures will be removed.

Reply:
Ok, it makes sense: it works for `numba` because it can recast them not as objects.

Actually, I'd have expected it to work even for `int`s, since even integers are singleton in Python (there should exist only one `1`):
```py
i = 1
j = 2 - 1
assert i is j
```
But maybe it's just `numba`'s fault once more: the moment they become `numba` integers, they are not any longer Python objects, so it's not safe to use `is` any longer.

Better to keep always `==` inside `numba`.

Reply:
If all occurrences of `==` are inside `numba` functions, mark this suggestion as solved.